includes/Modules/Analytics/Google_Service_AnalyticsProvisioning.php

```diff
@@ -24,6 +24,7 @@ class Google_Service_AnalyticsProvisioning extends Google_Service_Analytics {
 	/**
 	 * Constructs the internal representation of the Analytics service.
 	 *
+	 * @since n.e.x.t
```

Comment:
There should be a blank line above param/return docs. I'll just add that really quick.

Reply:
Thanks :)